src/rpc/client.cpp

```diff
@@ -59,6 +59,7 @@ static const CRPCConvertParam vRPCConvertParams[] = {
     {"listtransactions", 3, "include_watchonly"},
     {"walletpassphrase", 1, "timeout"},
     {"getblocktemplate", 0, "template_request"},
+    {"getunsolvedblock", 0, "address"},
```

Comment:
Shouldn't it be 1? I'm not sure what that number does.

Reply:
Parameter index

Reply:
Actually, this shouldn't be there.

Reply:
It has to do with parameter conversion to non-strings. That explains why I was having weird JSON encoding errors w/o manually adding quotes.